content/en/docs/languages/net/exporters.md

```diff
@@ -5,60 +5,27 @@ weight: 50
 
 {{% docs/languages/exporters/intro dotnet %}}
 
-{{% alert title="Note" color="info" %}}
+### Dependencies {#otlp-dependencies}
 
-If you use the OpenTelemetry .NET Automatic Instrumentation for
-[automatic instrumentation](/docs/languages/net/automatic) you can learn how to
-setup exporters following the
-[Configuration Guide](/docs/languages/net/automatic/config)
+If you want to send telemetry data to an OTLP endpoint (like the
+[OpenTelemetry Collector](#collector-setup), [Jaeger](#jaeger) or
+[Prometheus](#prometheus)), you can choose between two different protocols to
+transport your data:
 
-{{% /alert %}}
+- HTTP/protobuf
+- gRPC
 
-## Console exporter
-
-The console exporter is useful for development and debugging tasks, and is the
-simplest to set up.
-
-```sh
-dotnet add package OpenTelemetry.Exporter.Console
-dotnet add package OpenTelemetry.Extensions.Hosting
-```
-
-If you're using ASP.NET Core, configure the exporter in your ASP.NET Core
-services:
-
-```csharp
-var builder = WebApplication.CreateBuilder(args);
-
-builder.Services.AddOpenTelemetry().WithTracing(b =>
-{
-    b.AddConsoleExporter()
-    // The rest of your setup code goes here too
-});
-```
-
-Otherwise, configure the exporter when creating a tracer provider:
-
-```csharp
-using var tracerProvider = Sdk.CreateTracerProviderBuilder()
-    .AddConsoleExporter()
-
-    // Other setup code, like setting a resource goes here too
-
-    .Build();
-```
-
-## OTLP endpoint
-
-To send trace data to an OTLP endpoint (like the [collector](/docs/collector) or
-Jaeger) you'll want to configure an OTLP exporter that sends to your endpoint.
+Start by installing the respective exporter package as a dependency for your
+project:
 
 ```sh
 dotnet add package OpenTelemetry.Exporter.OpenTelemetryProtocol
 dotnet add package OpenTelemetry.Extensions.Hosting
 ```
 
-If you're using ASP.NET Core, configure the exporter in your ASP.NET Core
+### Usage
+
+If you're using ASP.NET Core, configure the exporters in your ASP.NET Core
```

Comment:
since we are splitting this based on ASP.NET Core vs others, perhaps it is better to move the 
`dotnet add package OpenTelemetry.Extensions.Hosting` as well to here.

Reply:
Not sure what this means, since my knowledge about .NET is limited, can you provide me with some suggestions what needs to be changed?

Reply:
(or can this be fixed in a follow up PR?)

Reply:
The usage section here starts with "if you are using ASP.NET Core".. And then later there is an "else" branch. It basically means, the doc is effectively split between asp.net core and non-asp.netcore. But the contents are following this principle fully. 
Specific example in this: `OpenTelemetry.Extensions.Hosting` package is only for Asp.Net Core apps. (technically any apps that supports Host model, that include worker apps too). So my suggestion was to move that package mention into the "If asp.net core" part.

Reply:
I agree with @cijothomas here. The hosting extensions package is almost exclusively relevant for ASP.NET Core apps.

Reply:
I understand, I can rewrite this

Reply:
please take another look